bundles/org.openhab.persistence.jdbc/src/main/java/org/openhab/persistence/jdbc/db/JdbcBaseDAO.java

```diff
@@ -618,12 +622,6 @@ public String getItemType(Item i) {
                         "JDBC::getItemType: Cannot detect ItemType for {} because the GroupItems' base type isn't set in *.items File.",
                         i.getName());
                 item = ((GroupItem) i).getMembers().iterator().next();
-                if (item == null) {
-                    logger.debug(
-                            "JDBC::getItemType: No ItemType found for first Child-Member of GroupItem {}, use ItemType for STRINGITEM as Fallback",
-                            i.getName());
-                    return def;
-                }
```

Comment:
`next()` will throw `NoSuchElementException` if the iteration has no more elements. So I don't think it can be null after being assigned in the line above?

Reply:
No need to handle this exception ?

Reply:
In this case it would be an existing bug as I simply removed dead code. Nevertheless, in case this method could be called with a group item having no children (hard for me to assess), I have now fixed this.